brainreg_napari/sample_data.py

```diff
@@ -0,0 +1,42 @@
+import zipfile
+from typing import List
+
+import numpy as np
+import pooch
+from napari.types import LayerData
+from skimage.io import imread
+
+# git SHA for version of sample data to download
+data_commit_sha = "72b73c52f19cee2173467ecdca60747a60e5fb95"
+
+POOCH_REGISTRY = pooch.create(
+    path=pooch.os_cache("brainreg_napari"),
+    base_url=(
+        "https://gin.g-node.org/cellfinder/data/"
+        f"raw/{data_commit_sha}/brainreg/"
+    ),
+    registry={
+        "test_brain.zip": "7bcfbc45bb40358cd8811e5264ca0a2367976db90bcefdcd67adf533e0162b5f"  # NoQA
```

Comment:
I think it's better to be explicit here, and the docs seem to use `noqa` https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html
```suggestion
        "test_brain.zip": "7bcfbc45bb40358cd8811e5264ca0a2367976db90bcefdcd67adf533e0162b5f"  # noqa: E501
```

Reply:
What does the hash do here?

Reply:
It's the expected hash of the file - if the downloaded file doesn't match this hash, then `pooch` knows that the wrong file has been downloaded and can error.